src/datasources/asset/types.ts

```diff
@@ -1,11 +1,17 @@
 import { FieldDTO } from '@grafana/data'
 import { DataQuery } from '@grafana/schema'
 
-export interface AssetMetadataQuery extends DataQuery {
+export interface ListAssetsQuery extends DataQuery {
   workspace: string,
   minionIds: string[]
 }
 
+export interface CalibrationForecastQuery extends DataQuery {
+}
+
+export interface AssetSummaryQuery extends DataQuery {
```

Comment:
To keep this more organized, I'm thinking we can split this in 3 files:

ListAssets.types.ts
CalibrationForecastQuery.types.ts
AssetSummaryQuery.types.ts

What do you think ?

Reply:
We can keep in types.ts defined only the common declarations (such as AssetQuery)